src/objects/zcl_abapgit_object_clas.clas.abap

```diff
@@ -245,6 +308,27 @@ CLASS ZCL_ABAPGIT_OBJECT_CLAS IMPLEMENTATION.
     io_xml->add( iv_name = 'TPOOL'
                  ig_data = add_tpool( lt_tpool ) ).
 
+    IF io_xml->i18n_params( )-serialize_master_lang_only IS INITIAL.
```

Comment:
perhaps move this part to a separate method?

Reply:
A method to check if only master language is active?